### PR TITLE
Client meeting minutes for Nov 18

### LINKED ISSUE #84: Post Client Meeting Minutes for Nov. 18
**Describe what needs to get done**
Markdown file of the minutes from the client meeting

### PR DESCRIPTION
**Pull request by vnajack**
- closes #84 
- create Nov. 18 minutes
- branch update to ensure it contains everything so far

### CODE REVIEW

**Review by mkuelker — CHANGES_REQUESTED** [review]
Let's update this branch by merging dev onto meeting minutes.

**Comment by vnajack** [comment]
before this PR can proceed
- need to close PR #129 
- refresh this PR

**Review by mkuelker — APPROVED** [review]
Looks Good

File: docs/meetings/client/2022_11_18.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+- Showing their phoneme mappings process time: 1.72 words per second when mapping the phonemes from provided text
+- Next steps
+  - Integrating and optimizing Gentle
+  - Generating video
+
+## Team A
+
+- Demo available as well
+  - Mouth only
```
Comment (LoganParker) [inline docs/meetings/client/2022_11_18.md:53]: due to available images

File: docs/meetings/client/2022_11_18.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  - Team B
+  - Team C
+
+## Team B
+
+- Demo video of a character with lips syncing
+  - Body poses as well
+  - Eyebrows
+  - Eyes
+  - 30 frames per second
```
Comment (LoganParker) [inline docs/meetings/client/2022_11_18.md:32]: we could also do 30fps if desired

**Review by LoganParker — APPROVED** [review]
looks great! Thank you for taking these!!

**Comment by mkuelker** [comment]
Merging client meeting minutes into Development, since Veronica is currently out.

**Pull request merged**